### ISSUE TITLE
ether_type value in ethernet header configuration is set to default instead of the value provided in configuration

### ISSUE THREAD

**Issue by ashutshkumr** [open-traffic-generator/ixia-c#64]
The ether_type value set by the user is `2048(0x0800)`, but the value is defaulting to `65535(0xFFFF)` on the wire

Test:
```import snappi


def test_plvision_issue(api):
    api = snappi.api(location="https://127.0.0.1:443", ext="ixnetwork")
    config = api.config()

    tx, rx = config.ports.port(
        name="tx", location="otg-novus100g.lbj.is.keysight.com;1;5"
    ).port(name="rx", location="otg-novus100g.lbj.is.keysight.com;1;6")

    flow = config.flows.flow(name="f1")[-1]
    flow.tx_rx.port.tx_name = tx.name
    flow.tx_rx.port.rx_name = rx.name

    f = config.flows[0]
    f.packet.ethernet().custom()
    eth, payload = f.packet[0], f.packet[1]
    eth.src.value = "00:CD:DC:CD:DC:CD"
    eth.dst.value = "00:AB:BC:AB:BC:AB"
    eth.ether_type.value = 2048

    payload.bytes = "45000056006600004006af92c0a800010a00000104d20050000000000000000050022000c3d50000000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f202122232425262728292a2b2c2d"

    print(config)
    api.set_config(config)```

**Comment by ankur-sheth** [comment]
@shramroy can you please close this issue if it has been resolved.

**Comment by shramroy** [comment]
The fix is available from the Ixia-c release v0.0.1-2738.
